fix(derailed/k9s) set version constraint

CrystalMethod opened the pull request.

CrystalMethod committed:
fix(derailed/k9s) set version constraint

CrystalMethod committed:
Merge branch 'main' into fix/k9s-arch-pattern

CrystalMethod commented:
#9435

suzuki-shunsuke commented:
Thank you always!

suzuki-shunsuke reviewed (commented):
When you add `version_constraint`, please add a version in pkg.yaml too.
https://github.com/aquaproj/aqua-registry/blob/main/CONTRIBUTING.md#test-multiple-versions

suzuki-shunsuke committed:
fix(derailed/k9s): add test

suzuki-shunsuke committed:
refactor(derailed/k9s): make version_constraint simple

https://aquaproj.github.io/docs/reference/registry-style-guide/#version_overrides-style-guide
